### PR TITLE
SWC-6572

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Couple of targeted css fixes.
Results in the following output:
Screen Shot 2023-09-21 at 5 08 39 PM
Screen Shot 2023-09-21 at 5 15 27 PM

File: src/main/webapp/sass/_core.scss [hunk 1]
```diff
@@ -3299,6 +3304,10 @@ div.border {
   transition: opacity 0.3s ease-in;
 }
 
+#body .SRC-grid-item .UserBadge {
+  width: 100%;
+}
```
Comment (jay-hodgson) [inline src/main/webapp/sass/_core.scss:3309]: Let the grid item parent determine the width.

File: src/main/webapp/sass/_core.scss [hunk 2]
```diff
@@ -2592,9 +2600,6 @@ Looks for any child element under .modal-fullscreen that has a class that contai
 
 #body .inline-user-badge {
   display: inline-flex;
-  div {
-    display: inline;
```
Comment (jay-hodgson) [inline src/main/webapp/sass/_core.scss:2596]: display inline-flex

File: src/main/webapp/sass/_core.scss [hunk 3]
```diff
@@ -2592,9 +2600,6 @@ Looks for any child element under .modal-fullscreen that has a class that contai
...
-  }
   .SRC-userImgSmall,
   .SRC-userImgMedium,
   .SRC-userImgLarge,
```
Comment (nickgros) [inline src/main/webapp/sass/_core.scss:2605]: These classes are no longer used

**Review by nickgros — COMMENTED** [review]
Thanks @jay-hodgson !

**Pull request merged**